plat/kvm/arm/setup.c

```diff
@@ -233,11 +230,6 @@ void __no_pauth _ukplat_entry(struct ukplat_bootinfo *bi)
 	if (unlikely(rc < 0))
 		UK_CRASH("Failed to get PSCI method: %d.\n", rc);
 
-#if CONFIG_RTC_PL031
-	/* Initialize RTC */
-	pl031_init_rtc(fdt);
```

Comment:
Since now the `fdt` variable is only used by SMP you can also mark it as `__maybe_unused`

Reply:
I noticed that `fdt` is passed every time to `kvm_console_init()`, therefore I don't think it should be marked as `__maybe_unused` or removed.